packages/terra-data-grid/tests/jest/ColumnResizeHandle.test.jsx

```diff
@@ -40,7 +65,131 @@ describe('ColumnResizeHandle', () => {
 
     // Validate mock function was called from simulated onMouseDown event
     expect(mockResizeMouseDown).toHaveBeenCalled();
+  });
 
-    expect(wrapper).toMatchSnapshot();
+  it('executes the callback function on MouseUp', () => {
+    const mockResizeMouseUp = jest.fn();
+    const wrapper = mountWithIntl(
+      <ColumnResizeHandle
+        onResizeMouseUp={mockResizeMouseUp}
+      />,
+    );
+    const resizeHandle = wrapper.find('.resize-handle');
+    // Simulate onMouseDown event
+    resizeHandle.invoke('onMouseUp')();
+
+    // Validate mock function was called from simulated onMouseDown event
+    expect(mockResizeMouseUp).toHaveBeenCalled();
+  });
+
+  it('sets widths if isActive is set to be true', () => {
+    const wrapper = mountWithIntl(
+      <ColumnResizeHandle
+        columnIndex={0}
+        columnText="Vitals"
+        columnWidth={200}
+        isActive
+        height={200}
+        minimumWidth={60}
+        maximumWidth={300}
+      />,
+    );
+
+    expect(wrapper.find('.resize-handle')).toHaveLength(1);
+    expect(wrapper.find('.resize-handle').props()['aria-valuenow']).toBe(200);
+    expect(wrapper.find('.resize-handle').props()['aria-valuemin']).toBe(60);
+    expect(wrapper.find('.resize-handle').props()['aria-valuemax']).toBe(300);
+  });
+
+  it('sets aria-label on focus', async () => {
+    const wrapper = mountWithIntl(
+      <ColumnResizeHandle />,
+    );
+
+    wrapper.find('.resize-handle').simulate('focus');
+
+    expect(wrapper.find('.resize-handle').props()['aria-label']).toBe('Terra.worklist-data-grid.resize-handle-template');
```

Comment:
We're testing for the bug value?

Reply:
Correct, that should be the default value when no locale is set.

Reply:
This is the i18n key rather than the string it's supposed to look up. It shouldn't be the aria-label.